server/task_executor_schema_update_ghost_sync.go

```diff
@@ -269,12 +281,12 @@ func executeGhost(task *api.Task, startedNs int64, statement string, waitSync *s
 
 	go func(ctx context.Context, migrationContext *base.MigrationContext, waitSync *sync.WaitGroup) {
```

Comment:
If `migrator.Migrate()` fails, this goroutine will be leaked.
We should explicitly create and cancel the context in `runGhostMigration` on receiving from error or done channel.

Reply:
> If `migrator.Migrate()` fails, this goroutine will be leaked. We should explicitly create and cancel the context in `runGhostMigration` on receiving from error or done channel.

Line 279 `defer cancel()`, so there should be no leakage, IMO.